### PR TITLE
package changes to support node19

### PR DESCRIPTION
**Pull request by bueche**
By submitting a PR to this repository, you agree to the terms within the [Code of Conduct](https://github.com/join-monster/join-monster/blob/master/CODE_OF_CONDUCT.md). Please see the [contributing guidelines](https://github.com/join-monster/join-monster/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

Currently join-monster only supports some earlier versions of node 14x - 16x. The sqllite3 install fails for node 17-19.
This PR is meant to add those additional versions of node.

### References

> Include any links supporting this change such as a:
>
> - GitHub Issue/PR number addressed or fixed
> - StackOverflow post
> - Related pull requests/issues from other repos
>
> If there are no references, simply delete this section.

### Testing

> Describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. If this library has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors.
>
> Please include any manual steps for testing end-to-end or functionality not covered by unit/integration tests.
>
> Also include details of the environment this PR was developed in (language/platform/browser version).

- [x] This change adds test coverage for new/changed/fixed functionality

### Checklist

- [x] I have added documentation for new/changed functionality in this PR via comments and by updating the change log

### CODE REVIEW

**Comment by bueche** [comment]
the linting workflow step failed due to some issue with the github workflow itself. it was not picking up the latest workflow code. I had modified the workflow code because it was not picking up the branch before loading the dependencies. The new dependencies needed the new package.json for linting.

File: test-api/server.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+/*
```
Comment (lorensr) [inline test-api/server.js:2]: why commented out?

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 #### Updated
 - [#495](https://github.com/join-monster/join-monster/pull/495): Added support for GraphQL v16_6_0. There were a number of breaking changes introduced in GraphQL v16 and these impacted a large number of tests and bit of the code. This provides fixes for those breaking changes. It also drops support for Node versions 10 - 13 but adds support for Node v16.
+- - [496](https://github.com/join-monster/join-monster/pull/496): Adding support for Node v17 - v19.
```
Comment (lorensr) [inline docs/CHANGELOG.md:5]: `- [#496]`

File: .github/workflows/ci-tests.yml [hunk 3]
```diff
@@ -34,8 +34,8 @@ jobs:
     strategy:
       matrix:
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-        # node-version: [10.x, 12.x, 13.x, 14.x, 15.x]
-        node-version: [14.x, 15.x, 16.x]
+        # covering 14, 16, 17 then 18 and 19. assuming 15 will work otherwise
```
Comment (lorensr) [inline .github/workflows/ci-tests.yml:37]: not sure what "otherwise" means here

File: test-api/server.js (hunk 1 above)
Comment (lorensr) [reply test-api/server.js:2]: ah I see: https://github.com/join-monster/join-monster/issues/497

**Pull request merged**